File: src/DefaultHighlighter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Drawing;

class DefaultHighlighter : Highlighter
{
    public DefaultHighlighter()
    {
        AddColor(TokenType.PlainText, 0xcbcdcf);
        AddColor(TokenType.SingleLineComment, 0xe04c55);
        AddColor(TokenType.StringLiteral, 0xb630ff);
        AddColor(TokenType.Parentheses, 0x31a0e0);
        AddColor(TokenType.CurlyBrackets, 0x459646);
        AddColor(TokenType.SquareBrackets, 0x459646);
        AddColor(TokenType.Semicolon, 0x459646);
        AddColor(TokenType.Colon, 0x459646);
        AddColor(TokenType.Digit, 0x8848ec);
        AddColor(TokenType.Operator, 0x31a0e0);

        AddColor(TokenType.Keyword, 0x0058d0);
        AddColor(TokenType.PrimitiveType, 0x0058d0);
        AddColor(TokenType.BuiltInType, 0x0058d0);
        AddColor(TokenType.Preprocessor, 0xffa13d);
    }
}

File: src/com/rb/edit/highlighting/DefaultHighlighter.java
package com.rb.edit.highlighting;

import com.rb.edit.Highlighter;

public class DefaultHighlighter extends Highlighter {
	public DefaultHighlighter() {
		setDefaultColor(0xcbcdcf);
		addColor("single-line-comment", 0xe04c55);
		addColor("string-literal", 0xb630ff);
		addColor("character-literal", 0xb630ff);
		addColor("bracket", 0x31a0e0);
		addColor("curly-brace", 0x459646);
		addColor("square-brace", 0x459646);
		addColor("semicolon", 0x459646);
		addColor("colon", 0x459646);
		addColor("digit", 0x8848ec);
		addColor("operator", 0x31a0e0);
	}
}
